pkg/utils/name_transformers.go

```diff
@@ -70,6 +70,6 @@ func MapDotNETJSONFormat(secrets map[string]string) map[string]string {
 
 // Test whether the file is ASP.NET Core settings file
 func IsDotNETSettingsFile(fileName string) bool {
-	matched, _ := regexp.MatchString(`(?i)appsettings(\.[a-z0-9-]+)?\.json`, fileName)
+	matched, _ := regexp.MatchString(`(?i)appsettings(\.[a-z0-9_-]+)?\.json`, fileName)
```

Comment:
You'll want to squash this commit into the previous commit. No need for a new commit if the code you're changing isn't yet in `main` :)